physionet-django/project/modelcomponents/publishedproject.py

```diff
@@ -206,16 +206,22 @@ def file_display_url(self, subdir, file):
         return reverse('display_published_project_file',
             args=(self.slug, self.version, os.path.join(subdir, file)))
 
-    def has_access(self, user):
+    def has_access_to_project_and_files(self, user):
```

Comment:
We can get more feedback on the names from the MIT crew, but I'd vote for something along the lines of `can_view_files` and `is_authorized` or something like that. We can move this discussion to the MIT repo.